src/build.rs

```diff
@@ -49,15 +51,88 @@ fn rustc_minor_version() -> Option<u32> {
     otry!(pieces.next()).parse().ok()
 }
 
-/// Ensure that `rustup` has the `wasm32-unknown-unknown` target installed for
-/// current toolchain
-pub fn rustup_add_wasm_target(step: &Step) -> Result<(), Error> {
-    let msg = format!("{}Adding the Wasm target...", emoji::TARGET);
-    PBAR.step(step, &msg);
+/// Get rustc's sysroot as a String
+fn get_rustc_sysroot() -> Result<String, FromUtf8Error> {
+    String::from_utf8(
+        Command::new("rustc")
+            .args(&["--print", "sysroot"])
+            .stdout(Stdio::piped())
+            .output()
+            .unwrap()
```

Comment:
This'll want to be sure to use `?` instead of `.unwrap()` here, and I think we'll also want to check `.status` to ensure it was successful before returning `Ok` as well

Reply:
Yes, good point! Thanks!